Tribler/Core/Modules/MetadataStore/serialization.py

```diff
@@ -0,0 +1,261 @@
+from __future__ import division
+
+import struct
+from datetime import datetime, timedelta
+
+from Tribler.Core.exceptions import InvalidSignatureException
+from Tribler.pyipv8.ipv8.deprecated.payload import Payload
+from Tribler.pyipv8.ipv8.keyvault.crypto import default_eccrypto
+from Tribler.pyipv8.ipv8.messaging.serialization import Serializer
+
+EPOCH = datetime(1970, 1, 1)
+INFOHASH_SIZE = 20  # bytes
+
+SIGNATURE_SIZE = 64
+EMPTY_SIG = '0' * 64
+
+crypto = default_eccrypto
+serializer = Serializer()
```

Comment:
There is also a `default_serializer` available. Actually, why redefine these here anyway?

Reply:
oops, overlooked that...